pootle/core/decorators.py

```diff
@@ -24,10 +24,10 @@
 from pootle_project.models import Project, ProjectSet, ProjectResource
 from pootle_store.models import Store
 from pootle_translationproject.models import TranslationProject
-from virtualfolder.helpers import extract_vfolder_from_path
+from virtualfolder.helpers import extract_vfolder_from_request
 
 from .exceptions import Http400
-from .url_helpers import split_pootle_path
```

Comment:
There is another usage of `split_pootle_path` in this file, so either get the import back, or replace it.